### PR TITLE
Modify the execution mode for host mitigations.

### PR DESCRIPTION
**Pull request by museQi**
Modify the execution mode for host mitigations.
change to run all cases.
- Verification run: http://openqa.qa2.suse.asia/tests/65011

### CODE REVIEW

**Comment by museQi** [comment]
@XGWang0 @zyuhu Please help me review.

**Comment by zyuhu** [comment]
LGTM

File: lib/Mitigation.pm [hunk 1]
```diff
@@ -330,7 +330,8 @@ sub check_sysfs {
                     update_tss_attr(file_name => "$junit_file", attr => 'failures', value => $total_failure_tc_count);
                     update_tss_attr(file_name => "$junit_file", attr => 'tests', value => $total_tc_count);
                     parse_junit_log("$junit_file");
-                    die "Fail on the $self->{sysfs}->{$value}->{$sysfs_name_item}"; }
```
Comment (XGWang0) [inline lib/Mitigation.pm:333]: Why need to remove "die", which will cause the program terminate when case is failed. is what you want?

Comment (museQi) [reply lib/Mitigation.pm:333]: Yes.I don't want to terminate the program once meeting failed case.So I remove die.
I want all the cases about mitigations can be executed.

Comment (XGWang0) [reply lib/Mitigation.pm:333]: OK, LGTM

**Comment by museQi** [comment]
@alice-suse Please help me review and merge.

**Pull request merged**